apps/src/templates/progress/ProgressTable.jsx

```diff
@@ -74,6 +82,7 @@ class ProgressTable extends React.Component {
               isSummaryView={isSummaryView}
               lessons={category.lessons}
               levelsByLesson={category.levels}
+              hidden={this.checkVisibility(category.lessons)}
```

Comment:
I realize this is a work in progress, but I don't see any `hidden` prop on ProgressGroup